cmd/testrunner.go

```diff
@@ -380,6 +383,13 @@ func testRunnerPipelineCommandAction(cmd *cobra.Command, args []string) error {
 		return fmt.Errorf("failed to read global config: %w", err)
 	}
 
+	esClientInfo, err := esClient.Info(ctx)
+	if err != nil {
+		return fmt.Errorf("fetching stack version failed: %w", err)
+	}
+
+	cmd.Println(version.Version())
+	cmd.Printf("elastic-stack: %s\n", esClientInfo.Version.Number)
```

Comment:
As this is printed by each test action, this is printed multiple times, once for each type of test, when running `elastic-package test` without specifying a subcommand. Is this intended?

Reply:
it is intentional, as the version from the stack is taken from the clients / profile.

would it make sense to create a esClient right at the begining of the test cmd to get the version?